lemarche/siaes/migrations/0073_siae_brevo_company_id.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.13 on 2024-05-13 12:29
+
+from django.db import migrations, models
+
+
+def populate_brevo_company_id(apps, schema_editor):
+    Siae = apps.get_model("siaes", "Siae")
+
+    for siae in Siae.objects.all():
+        if siae.extra_data.get("brevo_company_id"):
+            siae.brevo_company_id = siae.extra_data.get("brevo_company_id")
+            siae.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("siaes", "0072_siae_extra_data"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="siae",
+            name="brevo_company_id",
+            field=models.CharField(blank=True, max_length=80, null=True, verbose_name="Brevo company id"),
+        ),
+        migrations.RunPython(populate_brevo_company_id),
```

Comment:
```suggestion
        migrations.RunPython(populate_brevo_company_id, reverse_code=migrations.RunPython.noop),
```
ça permet de gérer les reverses de migration :)

Reply:
merci ! fixed ici https://github.com/gip-inclusion/le-marche/pull/1208/commits/e2aac5044141462c0bcabd7a0431cf3496bb0ba5